packages/picker/test/index.ts

```diff
@@ -286,6 +286,13 @@ export function runPickerTests(): void {
             expect(el.invalid).to.be.true;
             await expect(el).to.be.accessible();
         });
+        it('renders loading accessibly', async () => {
+            el.loading = true;
+            await elementUpdated(el);
+
+            expect(el.loading).to.be.true;
+            await expect(el).to.be.accessible();
```

Comment:
This test "is this generally accessible" which is great, but it doesn't test for a specific accessibility tree output. I think we want to do something like https://github.com/adobe/spectrum-web-components/blob/main/packages/picker/test/index.ts#L1136-L1147 and test for "Picker in loading state Loading...". This will ensure the `<button>` isn't `disabled` and that the expected labels are being applied.